assets/javascripts/discourse/initializers/discourse-post-event-decorator.js

```diff
@@ -201,9 +83,20 @@ function initializeDiscoursePostEventDecorator(api) {
       if (helper) {
         const post = helper.getModel();
 
-        if (post && post.event) {
-          _decorateEvent(api, cooked, post.event);
+        if (!post?.event) {
+          return;
         }
+
+        const div = document.createElement("div");
+        cooked.querySelector(".discourse-post-event").before(div);
+
+        helper.renderGlimmer(
+          div,
+          hbs`<DiscoursePostEvent @event={{@data.event}} />`,
```

Comment:
you could rename it to gjs and use `<template>` instead of `hbs` thingy (and then also use a local var instead of passing the args object explicitly)